stat/distmv/studentst.go

```diff
@@ -208,18 +206,18 @@ func studentsTConditional(observed []int, values []float64, nu float64, mu []flo
 // findUnob returns the unobserved variables (the complementary set to observed).
 // findUnob panics if any value repeated in observed.
```

Comment:
In order to maintain this invariant we need to check for repeats. This suggests an approach of sorting the `observed` slice and walking it, retaining gaps and panicking on repeats. Whether it's better to preallocate or append (or partial preallocate and then append) is unclear and would need measurement.

Can you commit the benchmark please (over a variety of `observed` lengths)?

Reply:
The godoc for the user-facing functions does not say the parameter will be mutated, so it is unsafe to sort and what you have is necessary.